templates/layouts/one-column-overlay.html.twig

```diff
@@ -18,7 +18,7 @@
 {% else %}
   {% set hero_body = content.overlay %}
   {% set hero_image = content.main %}
-  {% set modifier_class = settings.extra_classes %}
+  {% set modifier_class = ('jumpstart-ui--one-column-overlay ' ~ settings.extra_classes)|trim %}
```

Comment:
@pookmish I'm not sure how we can properly add a space after `jumpstart-ui--one-column-overlay` since without the space, the `settings.extra_classes` concats immediately after.

i.e. 
```<div class="su-hero jumpstart-ui--one-column-overlaysu-hero-height su-hero-left"> ```

Reply:
What you have looks correct. does this not render out the spaces correctly?

Reply:
It renders correctly as long as the space is added after `'jumpstart-ui--one-column-overlay '` like how I currently have it in this PR.

I'm just curious if there's a more elegant way to add a singular space than to just literally add it 😄

Reply:
not that i know of. g2g